docs/labs/lab03-configlet-management/config_search.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2022 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the COPYING file.
+
+from cvprac.cvp_client import CvpClient
+import ssl
+ssl._create_default_https_context = ssl._create_unverified_context
+import requests.packages.urllib3
+requests.packages.urllib3.disable_warnings()
+
+# Create connection to CloudVision
+clnt = CvpClient()
+clnt.connect(nodes=['cvp1'], username="username",password="password")
+
+def main():
+
+   print('Retrieving configlets ...')
+
+   inventory = clnt.api.get_inventory()
+   configlets = clnt.api.get_configlets_and_mappers()['data']['configlets']
+   generated_configlet_mappers = clnt.api.get_configlets_and_mappers()['data']['generatedConfigletMappers']
+   configlet_mappers = clnt.api.get_configlets_and_mappers()['data']['configletMappers']
```

Comment:
Why make the call the **get_configlets_and_mappers()** 3 times? You can make the call once and grab all necessary data from the response.